src/parquet/types.cc

```diff
@@ -49,17 +49,11 @@ std::string FormatValue(Type::type parquet_type, const char* val, int length) {
       break;
     }
     case Type::BYTE_ARRAY: {
-      const ByteArray* a = reinterpret_cast<const ByteArray*>(val);
-      for (int i = 0; i < static_cast<int>(a->len); i++) {
-        result << a[0].ptr[i] << " ";
-      }
+      result << val << " ";
```

Comment:
Min, Max statistics for BA and FLBA types have only the `bytes` and no `length` associated with them.